### ISSUE TITLE
Zip File size

### ISSUE THREAD

**Issue by MattReimer** [joshtkehoe/lambda-python-gdal#2]
I'm having an issue where, when I compile GDAL with static Proj4 and numpy, as per your instructions, the final zip file is 57Mb which is too big for lambda. 

Have you found found any way of compiling it to be smaller and not lose the python bindings?

*BTW: these are great instructions!*

**Comment by AdventureBeard** [comment]
I wasn't able to get the package any smaller, but you can bypass the size requirement by uploading the zip from an S3 bucket rather than your machine. Not an ideal workaround, but it works!

**Comment by joshtkehoe** [comment]
Yep - you need to put the zip in a S3 bucket and then reference it from the Lambda. I believe the size limit there is 250mb.

**Comment by MattReimer** [comment]
Perfect. Thanks! I don't know why I assumed the S3 limit was the same. 

FYI:  While I was poking around I found the [marblecutter demo](https://github.com/mojodna/marblecutter-lambda/tree/master/deps) and managed to solve the "bigness" problem anyway. 

They have a prune file and it looks like they removed everything then put things back until it works. 

I managed to get my zip file down to 30Mb.  A bit finicky but it's good for keeping your code small.
